controller/contactusController.js

```diff
@@ -1,7 +1,28 @@
 // dbClient = require('../dbConnection.js');
 
-const contactus = {
-
-};
-
-module.exports = contactus;
\ No newline at end of file
+const contactus = async (req, res) => {
+    try {
+      const { name, email, message } = req.body;
+      if (!name) {
+        return res.status(400).send('Name is required');
+      }
+    
+      if (!email) {
+        return res.status(400).send('Email is required');
+      }
+  
+      if (!message) {
+        return res.status(400).send('Message is required');
+      }
+      console.log('Received data:', req.body);
```

Comment:
We should avoid adding in console.logs as much as possible

Reply:
Resolved